### ISSUE TITLE
Keymaps without modifiers

### ISSUE THREAD

**Issue by Ordoviz** [ifreund/river#39]
I would like to define shortcuts to change volume/brightness/take a screenshot that do not require holding a modifier. I propose adding a NULL modifier:

    riverctl map normal NULL XF86MonBrightnessDown spawn brillo -qU 5

**Comment by ifreund** [comment]
This is definitely a valid use case so river should support it. Adding a `NULL` modifier would certainly work. Other options would include allowing an empty string for the modifiers argument or combining the modifiers and keys arguments.
```
riverctl map normal "" XF86MonBrightnessDown spawn brillo -qU 5
```
or
```
riverctl map normal Mod4+Shift+Return spawn alacritty
```

**Comment by Ordoviz** [comment]
Combining the modifiers and keys arguments is what Sway does. Choosing this syntax would make it a little easier to convert your Sway config to river but it would also be a breaking change. I assume that the combined syntax is a little harder to parse. I let you decide.

**Comment by ifreund** [comment]
Fixed by 2b10720366a28b1de124c098e13317b26eff981b

I decided to go with the `""` option as I think it's the most consistent. I'd rather not combine the modifiers and keys arguments as I don't intend to allow bindings with multiple keys and the separation keeps things simpler to parse and imo more logical to read.
